inspirehep/modules/workflows/config.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Workflows configuration."""
+
+from __future__ import absolute_import, division, print_function
+
+
+#
+# Configuration used when matching references.
+#
+
+WORKFLOWS_REFERENCE_MATCHER_DEFAULT_CONFIG = {
+    'algorithm': [
+        {
+            'queries': [
+                {
+                    'path': 'reference.arxiv_eprint',
+                    'search_path': 'arxiv_eprints.value.raw',
+                    'type': 'exact',
+                },
+                {
+                    'path': 'reference.dois',
+                    'search_path': 'dois.value.raw',
+                    'type': 'exact',
+                },
+                {
+                    'path': 'reference.isbn',
+                    'search_path': 'isbns.value.raw',
+                    'type': 'exact',
+                },
+                {
+                    'path': 'reference.report_numbers',
+                    'search_path': 'report_numbers.value.fuzzy',
+                    'type': 'exact',
+                },
+                {
+                    'paths': [
+                        'reference.publication_info.journal_title',
+                        'reference.publication_info.journal_volume',
+                        'reference.publication_info.artid',
+                    ],
+                    'search_paths': [
+                        'publication_info.journal_title.raw',
+                        'publication_info.journal_volume',
+                        'publication_info.artid',
```

Comment:
Can you check if we still need these searches to be distinct, given the following commit by @iulianav? https://github.com/inspirehep/inspire-next/commit/09b1255ad0146a0d124a1a93dd1b1fba8ed6d16c#diff-472ccc26aabcd94f4c20bce673a6e90d

Reply:
That should be the case. However, I do not see any keys named page_artid in Labs for the few records I have inspected. Am I missing something?

Reply:
This has to do with the thing I say often: "ES is not a JSON store". The `copy_to` directive **does not** enhance the JSON that you are sending as the source document, but instructs ES to index the field in another inverted index; so when you retrieve records from ES you get the source JSON back, which hasn't changed (in particular it doesn't contain `page_artid`), but searches against that field in the inverted index will work.

Reply:
Ok, nice. I'll verify and make the necessary changes.

Reply:
I checked this and the records with publication_info no longer match.

Reply:
didn't we want to search for both `artid` and `page_start` in `page_artid` instead of doing `artid -> artid`, `page_start -> page_start` only?

Reply:
it's not about reducing the number of queries, but having more matches, when we wrongly identify what is a `page_start` and what is an `artid`.